Update community.rst from #3084 (#7003)
WeblateOrg/weblate: docs/devel/community.rst

Building translators community
==============================

> Following these recommendations support the creation of a full, multilingual post-editing tool. Good translations are defined through the systemic-functional model of House which aims at a contextual correct translation. Write your own `post-editing guide <https://www.taus.net/academy/best-practices/postedit-best-practices/machine-translation-post-editing-guidelines>`_ and alter these recommendations to fit your own definitions. In most cases the `browser-plugin of languageTool <https://www.languagetool.org/#firefox_chrome>`_  is useful as the proof-reading tool.

Reviewer: ```suggestion
Following these recommendations support the creation of a full, multilingual post-editing tool. Good translations are defined through the systemic-functional model of House which aims at a contextual correct translation. Write your own `post-editing guide <https://www.taus.net/academy/best-practices/postedit-best-practices/machine-translation-post-editing-guidelines>`_ and alter these recommendations to fit your own definitions. In most cases the `browser-plugin of languageTool <https://languagetool.org/#firefox_chrome>`_  is useful as the proof-reading tool.
```

The degree of quality of a translation is based on the degree of equivalence, the correspondence between the text profile and the text function. Because it cannot be calculated automatically, sufficient information should be collected to enable a uniform human evaluation. The two main parameters of agreement in a corresponding model are the macro-context - i.e. embedding in a larger social and literary context - and the micro-context consisting of field, tenor and mode.

Sources
-------
1. Marina Sanchez-Torron and Philipp Koehn in Machine Translation Quality and Post-Editor Productivity, Figure 1: https://www.cs.jhu.edu/~phi/publications/machine-translation-quality.pdf
2. Joanna Best und Sylvia Kalina.Übersetzen und Dolmetschen: eine Orientierungs-hilfe. A. Francke Verlag Tübingen und Base, 2002. Möglichkeiten der Übersetzungskritik S.101 und Folgende
3. post-editing guide, Sharon O'Brien and Fred Hollowood in MT Post-editing Guidelines, 2010 https://www.taus.net/academy/best-practices/postedit-best-practices/machine-translation-post-editing-guidelines

> 4. neural fuzzy repair, Bram Bulté and Arda Tezcan in Neural Fuzzy Repair: Integrating Fuzzy Matches into Neural MachineTranslation, 2019 https://www.aclweb.org/anthology/P19-1175.pdf

Reviewer: ```suggestion
4. neural fuzzy repair, Bram Bulté and Arda Tezcan in Neural Fuzzy Repair: Integrating Fuzzy Matches into Neural MachineTranslation, 2019 https://aclanthology.org/P19-1175.pdf
```

localization process easy for community translators.

.. image:: /screenshots/guide.png

Terminology management

> ---------------------

Reviewer: ```suggestion
----------------------
```

The degree of quality of a translation is based on the degree of equivalence, the correspondence between the text profile and the text function. Because it cannot be calculated automatically, sufficient information should be collected to enable a uniform human evaluation. The two main parameters of agreement in a corresponding model are the macro-context - i.e. embedding in a larger social and literary context - and the micro-context consisting of field, tenor and mode.

Sources
-------
1. Marina Sanchez-Torron and Philipp Koehn in Machine Translation Quality and Post-Editor Productivity, Figure 1: https://www.cs.jhu.edu/~phi/publications/machine-translation-quality.pdf
2. Joanna Best und Sylvia Kalina.Übersetzen und Dolmetschen: eine Orientierungs-hilfe. A. Francke Verlag Tübingen und Base, 2002. Möglichkeiten der Übersetzungskritik S.101 und Folgende

> 3. Translation definition, Kalle Hilsenbek, Gruppenbasierte Umgebung für mehrsprachige Übersetzungsquellen, 7.7 Übersetzungskritik, 2017: http://multitranslation.space/thesis
> 4. post-editing guide, Sharon O'Brien and Fred Hollowood in MT Post-editing Guidelines, 2010 https://www.taus.net/academy/best-practices/postedit-best-practices/machine-translation-post-editing-guidelines
> 5. neural fuzzy repair, Bram Bulté and Arda Tezcan in Neural Fuzzy Repair: Integrating Fuzzy Matches into Neural MachineTranslation, 2019 https://www.aclweb.org/anthology/P19-1175.pdf

Reviewer: Remove non-existing document:

```suggestion
3. post-editing guide, Sharon O'Brien and Fred Hollowood in MT Post-editing Guidelines, 2010 https://www.taus.net/academy/best-practices/postedit-best-practices/machine-translation-post-editing-guidelines
4. neural fuzzy repair, Bram Bulté and Arda Tezcan in Neural Fuzzy Repair: Integrating Fuzzy Matches into Neural MachineTranslation, 2019 https://www.aclweb.org/anthology/P19-1175.pdf
```
